test/libsolidity/syntaxTests/viewPureChecker/inline_assembly_instructions_disallowed.sol

```diff
@@ -0,0 +1,28 @@
+contract C {
+    function f() public {
+        assembly {
+            datasize(0)
+            dataoffset(0)
+            datacopy(0, 1, 2)
+            setimmutable(0, "x", 1)
+            loadimmutable("x")
+            linkersymbol("x")
+            memoryguard(0)
+            verbatim_1i_1o(hex"600202", 0)
+
+            pop(msize())
+            pop(pc())
+        }
+    }
+}
+// ----
+// SyntaxError 6553: (47-362): The msize instruction cannot be used when the Yul optimizer is activated because it can change its semantics. Either disable the Yul optimizer or do not use the instruction.
```

Comment:
How does this actually work :-)? We only run these tests with enabled yul optimizer? I wouldn't have guessed :-).

Reply:
Apparently. I was surprised too since I usually run `isoltest` without the `--optimize` flag.